github/compliance/model.py

```diff
@@ -67,10 +67,12 @@ class ScanState(enum.Enum):
     '''
     indicates the scan outcome of a scan (regardless of yielded contents).
 
-    SUCCEEDED: scan succeeded without errors (but potentially with findings)
-    FAILED:    scan failed (which typically implies there are not scan results)
+    SUCCEEDED:  scan succeeded without errors (but potentially with findings)
+    SKIPPED:    No scan took place (usually due to unchanged scanning environment)
```

Comment:
the hint (in parentheses) is misleading. we will skip scans not because of "unchanged scanning env", but because we found an earlier scan that has not yet been invalidated